addons/web_editor/static/src/js/editor/editor.js

```diff
@@ -149,7 +149,7 @@ var EditorMenuBar = Widget.extend({
         this.trigger_up('ready_to_save', {defs: defs});
         return $.when.apply($, defs).then(function () {
             self.snippetsMenu.cleanForSave();
-            return self._saveCroppedImages();
+            return self.rte._summernoteManager._saveCroppedImages(self.rte.editable());
```

Comment:
Instead of checking the private `_summernoteManager`, create a public method in the `rte` include where `_summernoteManager` is instantiated. So that here, you can do `self.rte.saveCroppedImages(` and in there you can do `this._summernoteManager.saveCroppedImages(`

EDIT: see next message.